<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AlarmWidget</class>
 <widget class="QWidget" name="AlarmWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>401</width>
    <height>116</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string notr="true">Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="frame">
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QLabel" name="timeLabel">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="font">
           <font>
            <pointsize>30</pointsize>
           </font>
          </property>
          <property name="text">
           <string notr="true">00:00</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="snoozeLabel">
          <property name="text">
           <string notr="true">Snoozed</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="repeatsLabel">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="text">
           <string notr="true">TextLabel</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="tSwitch" name="activeSwitch">
        <property name="text">
         <string notr="true">ActiveSwitch</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>tSwitch</class>
   <extends>QPushButton</extends>
   <header location="global">tswitch.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
